### PR TITLE
11266 room type manage

### PR DESCRIPTION
**Pull request by ahxtah1996**
https://edu-redmine.sun-asterisk.vn/issues/11266

### CODE REVIEW

**Comment by ahxtah1996** [comment]
ready

File: resources/views/admin/cinema/room_type.blade.php [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    $('#createRoomType').click(function () {
+        $('#saveBtn').val("create-room_type");
+        $('#room_type_id').val('');
+        $('#roomTypeForm').trigger("reset");
+        $('#modelHeading').html("{{ __('label.createRoomType') }}");
+        $('#ajaxModel').modal('show');
+    });
+    $('body').on('click', '.editRoomType', function () {
+        var room_type_id = $(this).data('id');
+        $.get("{{ route('room_type.index') }}" +'/' + room_type_id + '/edit', function (data) {
```
Comment (BunnyPi04) [inline resources/views/admin/cinema/room_type.blade.php]: convention

Comment (BunnyPi04) [reply resources/views/admin/cinema/room_type.blade.php]: chưa fix này

**Comment by ahxtah1996** [comment]
ready

**Pull request merged**